index.html

```diff
@@ -144,45 +188,50 @@ <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
             <div class="sectiondaily">
                 <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
             </div>
+            <p class="section2"> A smart assistant to make driving more safe, efficient, and fun by unlocking your most
+                 expensive computer: your car. </p>
+            
             <div class="skil">
                 <ul class="program1">
                     <li class="Program-skilhtml">html</li>
+                    <li class="Ruby">Ruby on rails</li>
                     <li class="Program-skilhtml">css</li>
                     <li class="skiljava">javascript</li>
                 </ul>
             </div>
         <div class="buttonsee">
             <button class="button1"> See Project</button>
             </div>
-
-        </section>
-        
-      
+        </div>
+    </section>
 </section>
-<section class="AboutcardBack">
-    <div >
-        <h2  class="tonic4" >About Myself </h2>
-        <p class="section1"> Hello I’m a software developer! I can help you build a product , 
-        feature or website Look through some of my work and experience! If you like what you see 
-        and have a project you need coded, don’t hestiate to contact me. </p>
-    </div>
-    <div class = "social-section">
-        <div class="connectmywork">
-            <h4 > LET’S CONNECT</h4>
+<section class="AboutcardBack" id="aboutcardid">
```

Comment:
Kindly fix the design and use flex and make sure your web page responding properly for different screen sizes. Right now, it seems About Myself self is just filling the half of its container.
Figma template:
<img width="674" alt="Screenshot 2022-12-16 at 10 00 30 PM" src="https://user-images.githubusercontent.com/25887253/208175399-ac78b597-d886-4a07-a3d8-be717d2b5022.png">

Your page:
<img width="1440" alt="Screenshot 2022-12-16 at 9 59 19 PM" src="https://user-images.githubusercontent.com/25887253/208175248-bf661c59-47d8-4458-8160-17d7cf51b0fc.png">